src/index.html

```diff
@@ -2,12 +2,174 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>MA-task-uber-eats</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap&subset=cyrillic" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
+    <header class="header" id="header">
+
+      <h1><a href="#" title="main page uber eats"><img src="./images/logo.png" alt="logo uber eats" class="header__image"></a></h1>
```

Comment:
Не создавай такую вложенность в одну строку
```suggestion
      <h1>
        <a href="#" title="main page uber eats">
          <img src="./images/logo.png" alt="logo uber eats" class="header__image">
        </a>
      </h1>
```

Reply:
> Не создавай такую вложенность в одну строку

fixed